src/dev-app/column-resize/column-resize-demo-module.ts

```diff
@@ -12,17 +12,23 @@ import {MatExpansionModule} from '@angular/material/expansion';
 
 import {ColumnResizeHome} from './column-resize-home';
 import {
-  DefaultEnabledColumnResizeDemoModule,
-  DefaultEnabledColumnResizeFlexDemoModule,
-  OptInColumnResizeDemoModule,
+  DefaultEnabledColumnResizeExampleModule,
+  DefaultEnabledColumnResizeFlexExampleModule,
+  OptInColumnResizeExampleModule,
 } from '@angular/components-examples/material-experimental/column-resize';
 
 @NgModule({
   imports: [
     MatExpansionModule,
-    DefaultEnabledColumnResizeDemoModule,
-    DefaultEnabledColumnResizeFlexDemoModule,
-    OptInColumnResizeDemoModule,
+
+    // TODO(crisbeto): currently the column resize is set up so that users import different
+    // modules depending on whether they want resizig to be enabled by default. This forces us
+    // to keep each example in its own module and import it separately, otherwise the default
+    // enabled module will override the opt-in one. Once we refactor it so we only have one
```

Comment:
FWIW refactoring to only one module / set of directives hasn't been decided yet.

Reply:
It still feels like something we should do. I'd expect the first issue we'll get once this hits stable is that people would want a data binding for `disabled`.